internal/server/handler.go

```diff
@@ -219,12 +221,13 @@ func (s *Server) GetPropertyLabels(
 		return nil, err
 	}
 	jsonRaw, err := protojson.Marshal(resp)
-	// To make the API response backward compatible. This is to add the outer
-	// level `{\"data\":`
-	jsonRaw = jsonRaw[8 : len(jsonRaw)-1]
 	if err != nil {
 		return nil, err
 	}
+	// To make the API response backward compatible. This is to remove the outer
+	// level `{\"data\":`
+	jsonRaw = jsonRaw[8 : len(jsonRaw)-1]
```

Comment:
nit: just wondering, do we not need to skip the trailing "}" ?

Reply:
Oh, it's doing that. Added in the comments